dts/bindings/watchdog/ti,cc13xx-cc26xx-watchdog.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2022 Florin Stancu <niflostancu@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: CC13xx / CC26xx watchdog
+
+compatible: "ti,cc13xx-cc26xx-watchdog"
+
+include: base.yaml
+
+properties:
+    reg:
+      required: true
+
+    label:
+      required: true
```

Comment:
Remove label property. We are moving away from them.

Reply:
yes, I saw this inside git history but totally forgot to remove them.